### DISCUSSION TITLE
Python 2 support?

### DISCUSSION THREAD

**Discussion by Skraldefar** [letmaik/pyvirtualcam#58] [Q&A] [answered]
I am looking for a program that emulates a webcam from opencv. I have a problem that i cannot use python3 because of other modules that i am using. I can use python3 "import pyvirtualcam", but not python "import pyvirtualcam". Is this program only made for python 3?

**Comment by letmaik** [comment, marked as answer]
Absolutely Python 3 only. Don't use Python 2, it's not supported anymore, including security issues. https://www.python.org/doc/sunset-python-2/
